compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1211,3 +1211,12 @@ error.unsupported.primitive.type.reason=\
 
 error.method.signature.not.match=\
   '{ballerinax/java}'METHOD_SIGNATURE_DOES_NOT_MATCH ''{0}''
+
+warning.deprecated=\
+  ''{0}'' is deprecated
+
+error.deprecation.annotation.should.available=\
+  deprecation annotation should be available, since deprecation documentation is available
```

Comment:
Shall we reword this to something like this?

invalid `Deprecated` usage on a non-deprecated construct; annotation `deprecated` not found.  

@MaryamZi @pubudu91 @rdhananjaya thoughts?

Reply:
Shall we say

``invalid `Deprecated` documentation on a non-deprecated construct; `@deprecated` annotation not found.``

Reply:
```
invalid documentation: 'Deprecated' documentation only allowed on constructs annotated as deprecated using the '@deprecated' annotation
```

Reply:
```
invalid documentation: 'Deprecated' documentation only allowed on constructs annotated as '@deprecated'
```

Reply:
Can't we just say

``invalid documentation: 'Deprecated' documentation is only allowed on constructs annotated as '@deprecated'``

Reply:
Changed